nearit-ui/src/main/java/com/nearit/ui_bindings/utils/LoadImageFromURL.java

```diff
@@ -22,11 +22,20 @@
     @Nullable
     private ProgressBar progressBar;
 
+    private String url;
+    private boolean enableReload = false;
+
     public LoadImageFromURL(@Nullable ImageView imageView, @Nullable ProgressBar progressBar) {
         this.imageView = imageView;
         this.progressBar = progressBar;
     }
 
+    public LoadImageFromURL(@Nullable ImageView imageView, @Nullable ProgressBar progressBar, boolean enableReload) {
```

Comment:
call the 3 argument constructor, from the 2 argument constructor, with the default value

Reply:
The default value should be a psf constant

Reply:
done (for the single arg constructor too)